src/main/java/be/seeseemelk/mockbukkit/plugin/PluginManagerMock.java

```diff
@@ -895,28 +898,37 @@ public void unsubscribeFromPermission(@NotNull String permission, @NotNull Permi
 	@Override
 	public void subscribeToDefaultPerms(boolean op, @NotNull Permissible permissible)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		Map<Permissible, Boolean> map = this.defaultPermissionSubscriptions.computeIfAbsent(op, k -> new WeakHashMap<>());
+		map.put(permissible, true);
 	}
 
 	@Override
 	public void unsubscribeFromDefaultPerms(boolean op, @NotNull Permissible permissible)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		Map<Permissible, Boolean> map = this.defaultPermissionSubscriptions.get(op);
+
+		if (map == null)
+			return;
```

Comment:
```suggestion
		if (map == null)
		{
			return;
		}
```

Reply:
Are braces really needed here? I think it makes it a lot more cluttered since it's a fairly small method.

Reply:
I think we should either have braces or inline that return-

Reply:
Hmm... I'd rather inline it but I'm not a huge ran of that either since it's a bit harder at a glance to see that it's a return.

Reply:
I think we should ask @seeseemelk what he thinks

Reply:
Should be inline :eyes:

Reply:
Ok, this is stupid. Braces it is.